requests_toolbelt/_compat.py

```diff
@@ -18,11 +18,13 @@
     from requests.packages.urllib3 import fields
     from requests.packages.urllib3 import filepost
     from requests.packages.urllib3 import poolmanager
+    from requests.packages.urllib3.util import timeout
 except ImportError:
     from urllib3 import connection
     from urllib3 import fields
     from urllib3 import filepost
     from urllib3 import poolmanager
+    from urllib3.util import timeout
 
 if requests.__build__ < 0x020800:
     gaecontrib = None
```

Comment:
Out of curiousity, where do you find the build information. When I do `requests.__build__` on the latest version of requests (2.9.1) I get `133377`. Also curious what __build__ signifies.